internal/observability/call.go

```diff
@@ -483,12 +492,13 @@ func (c call) logStreamEvent(err error, success bool, succMsg, errMsg string, ex
 		ce = c.edge.logger.Check(lvl, errMsg)
 	}
 
-	fields := []zap.Field{
-		zap.String("rpcType", c.rpcType.String()),
+	fields := c.fields[:0]
```

Comment:
Can we do this in another PR? This seems to add more logs for streaming and is out of scope of this diff. (but I appreciated this fix though!)

Reply:
shouldn't we be adding callerProcedure logging by default for both unary and stream?

Reply:
Nevermind this comment, I read something else than `c.fields[:0]`